src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -183,13 +184,14 @@ public void getUsers(String query, String orderBy,
               try {
                 response.write(JSON_MAPPER.writeValueAsString(user), "UTF-8");
               } catch (JsonProcessingException e) {
-                logger.error("This error should never happen: " + e);
+                throw new UncheckedIOException(e);
```

Comment:
User json -> User in RMB, and then -> User json here again. How likely is the exception? But I am fine with re-throw.

Reply:
Let's say the exception could happen (I do not know how), where are we going to handle it after the re-throw? Should there be another layer of try/catch to handle it and close the connection properly?